parsers/sort-by/sort-by.spec.ts

```diff
@@ -10,7 +10,7 @@ describe('sort-by', () => {
     expect(JSON.stringify(libs._.sortBy(seeds().tokens, ['name']))).toEqual(JSON.stringify(result));
     return;
   });
-  it('Get tokens - apply parsers - wihtout options', async () => {
+  it('Get tokens - apply parsers - without options', async () => {
```

Comment:
This change shouldn't be here. Please, move it to another PR.

Reply:
Fixed ✅